bin/bootstrap

```diff
@@ -6,8 +6,8 @@
 
 set -e
 
-DIR=/bin/script
-source "$DIR/common"
+DIR=/usr/local/bin
```

Comment:
Perhaps these lines should be simplified to `source "/usr/local/bin/_common"` for cohesion?